### DISCUSSION TITLE
New call sign.

### DISCUSSION THREAD

**Discussion by arthollingsworth** [magicbug/Cloudlog#2728] [General]
I have a new vanity call sign and need to reflect that in Cloudlog. What is the best way to handle this?

**Comment by HB9HIL** [comment]
> I have a new vanity call sign and need to reflect that in Cloudlog. What is the best way to handle this?

Hello @arthollingsworth 

I needed to google what a vanity callsign is and I think I got an idea what it is. If the Vanity Callsign becomes your primary Call which identifies you the you should change the Call in your User Account to that Vanity Call. 

If the new Callsign is something you use from time to time but your primary call is still another one you need to create a Stationlocation for that Callsign. 

Could you just explain for Europeans what a Vanity Call is so I can see if I'm right with what I think it is? 😆

**Comment by HB9HIL** [comment]
PS: Just for the Logic

The Callsign in the Account represents the "Operator"

The Callsign in the Stationlocation is the Callsign which is sent out.

E.g. a Clubstation does have it's Club Callsign (Station) and it will be sent out by different Operators.
